src/Aspire.Hosting.Azure/Provisioning/Provisioners/BicepProvisioner.cs

```diff
@@ -114,9 +114,9 @@ public override async Task GetOrCreateResourceAsync(AzureBicepResource resource,
             existingResource.ResourceGroup is { } existingResourceGroup)
         {
             var existingResourceGroupName = existingResourceGroup is ParameterResource parameterResource
-                ? parameterResource.Name
+                ? parameterResource.Value
                 : (string)existingResourceGroup;
-            resourceGroup = await context.GetResourceGroup(existingResourceGroupName, resourceLogger, cancellationToken).ConfigureAwait(false);
+            resourceGroup = await context.Subscription.GetResourceGroupAsync(existingResourceGroupName, cancellationToken).ConfigureAwait(false);
```

Comment:
Does this handle if the resource group doesn't exist?

Reply:
Depends on what you mean by handle. Before it would fallback to the default group. Now, it'll fail and you'll see something like this in the logs:

```
Provisioning sbemulator...
 Error provisioning sbemulator.
Azure.RequestFailedException: Resource group 'rg-aspire-deploy-not-real' could not be found.
Status: 404 (Not Found)
ErrorCode: ResourceGroupNotFound

Content:
{"error":{"code":"ResourceGroupNotFound","message":"Resource group 'rg-aspire-deploy-not-real' could not be found."}}
```